ArkSavegameToolkitNet/ArkSaveGame.cs

```
@@ -126,6 +126,95 @@ public bool LoadEverything()
+                                cryoArchive.GetBytes(4);
+
+                                var dino = new GameObject(cryoArchive);
+                                dino.ObjectId = Objects.Count + cryoCreatureObjects.Count +1 ;
+                                dino.IsCryo = true;
+
+                                var statusobject = new GameObject(cryoArchive);
+                                statusobject.ObjectId = Objects.Count + cryoCreatureObjects.Count + 2;
+
+                                dino.loadProperties(cryoArchive, new GameObject(), 0, null);
+                                var statusComponentRef = dino.GetProperty<PropertyObject>("MyCharacterStatusComponent");
+                                statusComponentRef.Value.ObjectId = statusobject.ObjectId;
+                                dino.properties.Remove(ArkName.Create("MyCharacterStatusComponent"));
+                                dino.properties.Add(ArkName.Create("MyCharacterStatusComponent"), statusComponentRef);
+
+                                statusobject.loadProperties(cryoArchive, new GameObject(), 0, null);
+                                
+
+                                cryoCreatureObjects.Add(dino);
+                                cryoCreatureObjects.Add(statusobject);
+
+                                cryoVa.Dispose();
+                                cryoMmf.Dispose();
+
+                                //remove temp cryo data file
+                                File.Delete(filePath);
+                            }
+                            finally
+                            {
+
+                            }
+                            
+
+                        }
+                    }
+
+                    foreach (var cryoCreatureObject in cryoCreatureObjects)
+                    {
+                        Objects.Add(cryoCreatureObject);
+                    }
+
+                }
+                catch 
```

do not suppress exceptions without handling them in some manner. at a minimum you should be logging them.

what scenarios do you expect an exception to occur and should be ignored?